exasol/python_extension_common/deployment/language_container_deployer_cli.py

```diff
@@ -152,6 +153,9 @@ def secret_callback(ctx: click.Context, param: click.Option, value: Any):
 @click.option('--alter-system/--no-alter-system', type=bool, default=True)
 @click.option('--allow-override/--disallow-override', type=bool, default=False)
 @click.option('--wait_for_completion/--no-wait_for_completion', type=bool, default=True)
+@click.option('--extract-timeout-minutes', type=int, default=5)
```

Comment:
I would set the default at least to 10 minutes

Reply:
See next push.